cirq/protocols/json_serialization_test.py

```diff
@@ -291,6 +291,7 @@ def test_mutually_exclusive_blacklist():
     'ExpressionMap',
     'FSIM_GATESET',
     'Heatmap',
+    'InterHeatmap',
```

Comment:
I'd like to rename this to `InteractionHeatmap`

Reply:
No problem!